install/share/73certmap.ldif

```diff
@@ -0,0 +1,17 @@
+## IPA Base OID:
+##
+## Attributes:          2.16.840.1.113730.3.8.22.x
```

Comment:
Sure, my plan was to wait for final agreement on the PR before registering the OIDs.